### ISSUE TITLE
Request: publish with homebrew for macOS

### ISSUE THREAD

**Issue by xxyxxyxyx1** [lukassr/keep-dark-desktop#1]
[homebrew](brew.sh) is a terminal-based package manager for macOS. i've installed almost every program with homebrew, it makes things very easy to manage and update.

**Comment by lukassr** [comment]
Good idea, I'm looking into it. I'm working in a auto-update function with the releases of GitHub. I think it's more comfortable than brew. Thanks for the idea!
